src/main.js

```diff
@@ -8,7 +8,8 @@ const getLogger = () => {
 	} else {
 		// app environment - use Winston
 		const logger = new AppLogger({
-			level: process.env.CONSOLE_LOG_LEVEL || 'silly'
+			level: process.env.CONSOLE_LOG_LEVEL || 'silly',
+			colorize: process.env.CONSOLE_LOG_COLORIZED || true
```

Comment:
**issue**: if you set this environment variable to `false`, that will actually be the string `'false'`, which is truthy, so there won't actually be a way to disable colours. one option could be invert the logic and make the environment variable `CONSOLE_LOG_UNCOLORIZED` instead, or you could check `process.env.CONSOLE_LOG_COLORIZED !== 'false'`. i would prefer inverting the logic, since that makes the default behaviour clearer